kafka-client/src/main/java/dev/responsive/kafka/internal/utils/SessionUtil.java

```diff
@@ -121,6 +122,7 @@ public static MongoClient connect(
 
     MongoClientSettings settings = MongoClientSettings.builder()
         .applyConnectionString(new ConnectionString(connectionString))
+        .applyToConnectionPoolSettings(b -> b.maxConnectionIdleTime(60000, TimeUnit.MILLISECONDS))
```

Comment:
How long are the Antithesis test interruptions? I guess we should probably make this a config so we can tune it at will?